### ISSUE TITLE
Add "icons" to "Drop here to hide."

### ISSUE THREAD

**Issue by scootergrisen** [hackademix/noscript#62]
Suggest changing:

    Drop here to hide.

To:

    Drop icons here to hide.

I think its more clear for a new user what can be dropped.
Maybe remove the "." at the end.

**Comment by JeffJacobson** [comment]
I still don't understand what this feature does and couldn't find anything about it in the [documentation](https://noscript.net/usage/) nor in either of the [User Contributed Guides](https://noscript.net/usage/#usercontributed-guides).

(I am using the Firefox version, BTW.)

**Comment by thinkinginbinary** [comment]
Tried using this, feature makes 0 sense intuitively.  As a fan of UI/UX easter eggs I can see the value. Has anyone considered rebuilding Noscript in other languages?

**Comment by JeffJacobson** [comment]
> Has anyone considered rebuilding Noscript in other languages?

I have not, but by "language" do you mean a programming language or a human language like Spanish, German, Japanese, etc. (internationalization (i18n))?

**Comment by JeffJacobson** [comment]
I made an animated GIF showing how to use the feature. (I only just now figured out what I'm supposed to drag there.) Recorded using [ScreenToGif](https://github.com/NickeManarin/ScreenToGif/).

![Screen recording of dragging an icon](https://github.com/hackademix/noscript/assets/713754/03dd34e8-e7b2-45df-9ca0-32ee9d5c3187)

I hereby release the above animated GIF into the public domain. Anyone can use it for documentation or whatever they want without permission or credit.

**Comment by thinkinginbinary** [comment]
Certainly a programming language. I tried using the feature that way on mobile in two different browsers & it did not work.

**Comment by hackademix** [comment]
> Certainly a programming language. I tried using the feature that way on mobile in two different browsers & it did not work.

Correct, this feature works on desktop only. Feel free to open an issue for Android compatibility.

Thanks @JeffJacobson for your contribution, I'll probably use to add a section in the usage page.

As for this very issue, the rationale seems sound. I'm changing the string in English to "Drop here icons to hide", hoping the translators catch up soon.

**Comment by thinkinginbinary** [comment]
Ok, I will open an issue. That is a nice string change. Here could also be removed.

**Comment by jon-dyer** [comment]
This feature just straight up doesn't work, this won't fix it. Linux kde/sway.
